### ISSUE TITLE
The offscreen menu should be next in the tab order upon opening the menu

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org#366]
### Description of issue
When opening the offscreen menu, the first option "become a partner" is near the end of the tab order because the offscreen menu is located at the bottom of the DOM.

### Expected result
Upon opening the offscreen menu, I should be able to reach the first option "become a partner" by tabbing once.

### Actual result
I can't reach the first option until navigating through the entire page

### Steps to reproduce
1. On a tablet screen size, open the offscreen menu
2. Confirm the first option "become a partner" does not receive focus upon tabbing

```
Path:      /
Browser:   Chrome 92.0.4515.131 on Mac OS 10.15.7
Viewport:  1792 x 1009 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-359--dcdesignweek.netlify.app%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=92.0&resolution=1792x1009&speed=1&start=true&ref=netlify-source)

Submitted from [Netlify Deploy Preview #359](https://deploy-preview-359--dcdesignweek.netlify.app/)

**Comment by marcusrelacion** [comment]
@joshkimux I've removed the library for the offscreen menu to work on a simpler solution.
